Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Configurations/DeviceSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Aquality.Selenium.Core.Configurations;
+using Aquality.Selenium.Core.Logging;
+using Aquality.Selenium.Core.Utilities;
+using OpenQA.Selenium.Appium;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Aquality.Appium.Mobile.Configurations
+{
+    public class DeviceSettings : IDeviceSettings
+    {
+        private readonly ISettingsFile settingsFile;
+        private readonly string deviceKey;
+
+        public DeviceSettings(string deviceKey)
+        {
+            settingsFile = GetDevicesSettings();
+            this.deviceKey = deviceKey;
+        }
+
+        private ISettingsFile GetDevicesSettings()
+        {
+            var deviceProfileName = EnvironmentConfiguration.GetVariable("devicesProfile");
+            var devicesProfile = deviceProfileName == null 
+                ? "devices.json" 
+                : $"devices.{deviceProfileName}.json";
+            Logger.Instance.Debug($"Get devices settings from: {devicesProfile}");
+
+            var jsonFile = FileReader.IsResourceFileExist(devicesProfile)
+                ? new JsonSettingsFile(devicesProfile)
+                : new JsonSettingsFile($"Resources.{devicesProfile}", Assembly.GetCallingAssembly());
```

Comment:
why exactly do we use calling assembly here? Even "screensAssembly" is better to use, the calling assembly is the current assembly for private methods

Reply:
I dont' want to use `screensAssembly` here. So, I'll remove seconds line.